src/chrome/content/rules/BR.xml

```diff
@@ -5,6 +5,8 @@
 	<target host="story.br.de"/>
 	<target host="www.br.de"/>
 	<target host="www.brreisen.de"/>
+	<target host="br-ticket.de" />
+	<target host="www.br-ticket.de" />
```

Comment:
Error on `(www.\)?shop.br-ticket\.de`

Reply:
ping @numismatika

Reply:
indeed they messed up the correct chain there, will do the redirect to the ticket company myself then